src/org/usfirst/frc/team5957/robot/Robot.java

```diff
@@ -41,27 +46,29 @@
      */
     public void robotInit() {
 		oi = new OI();
-        autoChooser = new SendableChooser<Command>();
-        teleChooser = new SendableChooser<Command>();
-        
         driveTrain.init();
         
+        joystick = new SendableChooser<ControllerType>();
```

Comment:
This is a very important line. Leaving it out leads to many days of hair-pulling debugging.